frontend/src/metabase/setup/components/PreferencesStep.jsx

```diff
@@ -87,12 +87,12 @@ export default class PreferencesStep extends Component {
           <StepTitle title={stepText} circleText={String(stepNumber - 1)} />
           <form onSubmit={this.formSubmitted.bind(this)} noValidate>
             <div className="Form-field">
-              {t`In order to help us improve Metabase, we'd like to collect certain data about usage through Google Analytics.`}{" "}
```

Comment:
This will lead to untranslated strings in the maintenance version. Are we good with it?

Reply:
Good catch. I think we are not, let me revert these changes

Reply:
but then it wouldn't be correct 🤔

Reply:
You are right. I think that it's better to have untranslated strings than wrong legal information. WDYT?

Reply:
@mazameli what do you think? I'm wondering if we can request new translations for this feature unless it is not an overkill

Reply:
@cdeweyx fyi

Reply:
I'm not 100% sure I'm appreciating the full context here, but guiding principles are:

- Having correct legal information comes first.
- We can easily update translations if we need to for this: worst case we just upload it to POEditor and then use auto-translate for it.